build

```diff
@@ -368,21 +368,81 @@ docker_cleanup() {
-    EMQX_DOCKERFILE="${EMQX_DOCKERFILE:-deploy/docker/Dockerfile}"
+    local EMQX_BUILDER_VERSION="${EMQX_BUILDER_VERSION:-5.1-3}"
+    local EMQX_BUILDER_PLATFORM="${EMQX_BUILDER_PLATFORM:-debian11}"
+    local EMQX_BUILDER_OTP="${EMQX_BUILDER_OTP:-25.3.2-1}"
+    local EMQX_BUILDER_ELIXIR="${EMQX_BUILDER_ELIXIR:-1.14.5}"
+    local EMQX_BUILDER=${EMQX_BUILDER:-ghcr.io/emqx/emqx-builder/${EMQX_BUILDER_VERSION}:${EMQX_BUILDER_ELIXIR}-${EMQX_BUILDER_OTP}-${EMQX_BUILDER_PLATFORM}}
+    local EMQX_RUNNER="${EMQX_RUNNER:-${EMQX_DEFAULT_RUNNER}}"
+    local EMQX_DOCKERFILE="${EMQX_DOCKERFILE:-deploy/docker/Dockerfile}"
+    local PKG_VSN="${PKG_VSN:-$(./pkg-vsn.sh)}"
+    # shellcheck disable=SC2155
+    local VSN_MAJOR="$(echo "$PKG_VSN" | cut -d . -f 1)"
+    # shellcheck disable=SC2155
+    local VSN_MINOR="$(echo "$PKG_VSN" | cut -d . -f 2)"
+    # shellcheck disable=SC2155
+    local VSN_PATCH="$(echo "$PKG_VSN" | cut -d . -f 3)"
+    local SUFFIX=''
     if [[ "$PROFILE" = *-elixir ]]; then
-        PKG_VSN="$PKG_VSN-elixir"
+        SUFFIX="-elixir"
     fi
-    local default_tag="emqx/${PROFILE%%-elixir}:${PKG_VSN}"
-    EMQX_IMAGE_TAG="${EMQX_IMAGE_TAG:-$default_tag}"
+    local DOCKER_REGISTRY="${REGISTRY:-docker.io}"
+    local DOCKER_ORG="${DOCKER_ORG:-emqx}"
+    local EMQX_BASE_DOCKER_TAG="${DOCKER_REGISTRY}/${DOCKER_ORG}/${PROFILE%%-elixir}"
+    local default_tag="${EMQX_BASE_DOCKER_TAG}:${PKG_VSN}${SUFFIX}"
+    local EMQX_IMAGE_TAG="${EMQX_IMAGE_TAG:-$default_tag}"
+    local EDITION=Opensource
+    local LICENSE='Apache-2.0'
     ## extra_deps is a comma separated list of debian 11 package names
-    local extra_deps=''
+    local EXTRA_DEPS=''
     if [[ "$PROFILE" = *enterprise* ]]; then
-        extra_deps='libsasl2-2,libsasl2-modules-gssapi-mit'
+        EXTRA_DEPS='libsasl2-2,libsasl2-modules-gssapi-mit'
+        EDITION=Enterprise
+        LICENSE='(Apache-2.0 AND BSL-1.1)'
+    fi
+    # shellcheck disable=SC2155
+    local ISO_8601_DATE="$(date -u +"%Y-%m-%dT%H:%M:%SZ")"
+    # shellcheck disable=SC2155
+    local GIT_REVISION="$(git rev-parse HEAD)"
+    local DOCKER_BUILDX_ARGS=(
+       --build-arg BUILD_FROM="${EMQX_BUILDER}" \
+       --build-arg RUN_FROM="${EMQX_RUNNER}" \
+       --build-arg EMQX_NAME="${PROFILE}" \
+       --build-arg EXTRA_DEPS="${EXTRA_DEPS}" \
+       --build-arg PKG_VSN="${PKG_VSN}" \
+       --file "${EMQX_DOCKERFILE}" \
+       --label org.opencontainers.image.title="${PROFILE}" \
+       --label org.opencontainers.image.edition="${EDITION}" \
+       --label org.opencontainers.image.version="${PKG_VSN}" \
+       --label org.opencontainers.image.revision="${GIT_REVISION}" \
+       --label org.opencontainers.image.created="${ISO_8601_DATE}" \
+       --label org.opencontainers.image.source='https://github.com/emqx/emqx' \
+       --label org.opencontainers.image.url='https://github.com/emqx/emqx' \
+       --label org.opencontainers.image.documentation='https://www.emqx.io/docs/en/latest/' \
```

Comment:
Minor note: EE docs live in [docs.emqx.com](https://docs.emqx.com/en/enterprise/v5.1/).